.eslintrc

```diff
@@ -40,6 +45,31 @@
           }
         ]
       }
+    },
+    {
+      // Turn off react linting rules for every file
+      "files": ["*"],
+      "rules": {
+        "react/react-in-jsx-scope": "off",
+        "react/prop-types": "off",
+        "react/display-name": "off",
+        "react/jsx-key": "off",
+        "react-hooks/exhaustive-deps": "off",
+        "react-hooks/rules-of-hooks": "off"
+      }
+    },
+    {
+      // Activate the react linting rules for certain files
```

Comment:
I agree it's a good idea to allow current failures. But I'd suggest inverting this, so all files are enforced by default, and we list out the current files to exempt. I recognize that's probably a very long list, but it would help enforce the rules automatically for new files. There may also be a psychological incentive to chip away at the list of bad files.

There's some precedent for this approach:

- ERBLint exceptions in [#5083](https://github.com/18F/identity-idp/pull/5083/files#diff-09d428dfddfee8285e0a4c19ae57822ebb38b78c405a75e7f63b721f6fb00f7f) (better enforcement in [#5557](https://github.com/18F/identity-idp/pull/5557/files#diff-09d428dfddfee8285e0a4c19ae57822ebb38b78c405a75e7f63b721f6fb00f7f))
- TypeScript exceptions in [#5482](https://github.com/18F/identity-idp/pull/5482/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0)
- Yarn workspace README exceptions in [#7666](https://github.com/18F/identity-idp/pull/7666/files#diff-5f4a554cbcb878b399fc22201fbbefca97fd35240d6de35d0a34a24e6e38a2c7R134-R156)

Reply:
Thanks, that's really helpful. I implemented that. As I feared it's a long list of "don't lint this file", but as you said I do now have a -strong- desire to get rid of this list by fixing lint problems and enabling 👍

Reply:
Beautiful 👍